src/pages/About.js

```diff
@@ -36,137 +35,85 @@ let p3 = "Inclusivity is one of our founding principles. We believe that everyon
-                    <Text accent={accent3} blurb={blurb3}/>
-                </Col>
-                <Col className={"mission-statement"} style={{marginRight: "0"}}>
-                    <Text accent={accent4} blurb={blurb4}/>
-                </Col>
-            </Row>
-        )
-    }
-}
+// function DividedSection({img, blurb, blurb2, img_align}) {
+//     let ImageHalf = () => {
+//         return (
+//             <Col className="no-margin-padding">
+//                 <div className="image-half-container" style={{backgroundImage: "url(" + img + ")"}}/>
+//             </Col>
+//         )
+//     }
+//     let TextHalf = () => {
+//         return (
+//             <Col className="no-margin-padding">
+//                 <div className="text-half-container">
+//                     <p className={"body-text"}>{blurb}</p>
+//                     {/* if there is a second paragraph */}
+//                     { blurb2 ? <p className={"body-text"} style={{marginTop: "50px"}}>{blurb2}</p> : ""}
+//                 </div>
+//             </Col>
+//         )
+//     }
+//     // (img_align = true) => image on left side
+//     if (img_align) {
+//         return(
+//             <Row style={{padding: "0", margin: "0"}}>
+//                 <ImageHalf/>
+//                 <TextHalf/>
+//             </Row>
+//         )
+//     }
+//     else {
+//         return (
+//             <Row style={{padding: "0", margin: "0"}}>
+//                 <TextHalf/>
+//                 <ImageHalf/>
+//             </Row>
+//         )
+//     }
+// }
+//
+// function BlurbAndPicture({on_mobile, img, blurb, blurb2, img_align, img_above}) {
+//     let Image = () => {
+//         return (
+//             <Row className="image-half-container" style={{backgroundImage: "url(" + img + ")"}}/>
+//         )
+//     }
+//     let Text = () => {
+//         return (
+//             <Row className={"no-margin-padding"}>
+//                 <div className="text-half-container">
+//                     <p className={"body-text"}>{blurb}</p>
+//                     {/* if there is a second paragraph */}
+//                     { blurb2 ? <p className={"body-text"} style={{paddingTop: "10px"}}>{blurb2}</p> : ""}
+//                 </div>
+//             </Row>
+//         )
+//     }
+//     if (on_mobile) {
+//         if (img_above) {
+//             return (
+//                 <Col className={"no-margin-padding"}>
+//                     <Image/>
+//                     <Text/>
+//                 </Col>
+//             )
+//         }
+//         else {
+//             return (
+//                 <Col className={"no-margin-padding"}>
+//                     <Text/>
+//                     <Image/>
+//                 </Col>
+//             )
+//         }
+//     }
+//     else {
+//         return (
+//             <DividedSection img={img} blurb={blurb} blurb2={blurb2} img_align={img_align}/>
+//         )
+//     }
+// }
```

Comment:
There's a couple commented out components here, do you plan on bringing them back?

Reply:
I commented it out when I was creating a reusable component, must have forgotten to go back and delete the commented section. Feel free to remove it.